regression-tests/backends/geoip-master

```diff
@@ -26,6 +26,13 @@ domains:
       - ns: ns1.example.com
       - ns: ns2.example.com
       - mx: 10 mx.example.com
+    mixed_weight.geo.example.com:
+      - a:
+          content: "%ip4"
+          weight: 10
+      - txt:
+          content: text
+          weight: 10
```

Comment:
this test does not really test the functionality

Reply:
Could you be more specific on the "functionality" which should be tested and which is not?

Is there something unclear in the commit message where I introduce the test? If yes, how should I reword it to make it clearer?

My goal is to demonstrate we do not regress/retrigger #7051, and ensure that the ANY question answer both records, the A question return the A record and the the TXT return the TXT.
Do you think my test is covering this goal?
Do you think my test is enough?

Reply:
although not sure if it's even possible to test it well.